components/ai_chat/core/common/pref_names.cc

```diff
@@ -48,4 +52,13 @@ void RegisterLocalStatePrefs(PrefRegistrySimple* registry) {
   registry->RegisterDictionaryPref(kBraveChatPremiumCredentialCache);
 }
 
+bool IsPageContextEnabledInitially(const PrefService& prefs) {
+  auto* has_pref =
+      prefs.GetUserPrefValue(kBraveChatPageContextEnabledInitially);
+  if (has_pref) {
+    return has_pref->GetBool();
+  }
+  return base::FeatureList::IsEnabled(features::kPageContextEnabledInitially);
+}
```

Comment:
Do we really need both? It seems to me the feature flag could be superseded by the preference.
Could we remove the feature flag now that we have user pref in settings?
We could do an one time migration to update the preference to false if the feature flag was disabled.

Reply:
If we're getting rid of the flag, I don't think we'd even need to do a migration - at the moment its only controlled via Griffin or the command line.

I'm happy to remove this :smile:

Reply:
Okay, so I was actually wrong, we do expose a flag - I think I'm just going to remove it still - we don't generally worry too much about migrating off flags and I can't imagine too many people are using it.